karma.conf.js.tmpl

```diff
@@ -94,6 +94,12 @@ module.exports = function(config) {
 
     // Continuous Integration mode
     // if true, Karma captures browsers, runs the tests and exits
-    singleRun: false
+    singleRun: false,
+
+    client: {
+      jasmine: {
+        random: false
```

Comment:
Please add a TODO and a bug to re-enable this when possible, also explaining why it's not possible for now (basically your review comment).

Reply:
done